app/views/transaction/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<%= render "shared/navbar", {text:"Transaction", path:root_path} %>
+<%= render "shared/flash_messages" %>
+<div class="card" style="height:60px;">
+  <div class="card-body">
+    <div class="row">
+      <div class="col-7">
+        <p class="text-muted">Credit</p>
+      </div>
+      <div class="col-4">
+        <p class="text-dark fw-bold">
+          <small>
+            <% credit = 0 %>
+            <% @categories.each do |group| %>
+              <% total_groups = @categories.count %>
+              <% group.categories.each do |transaction| %>
+                <% credit += transaction.amount %>
+              <% end %>
+            <% end %>
+            <%= "$#{credit}" %>
+          </small>
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] The icons are not visible for some reason on the deployed version, kindly look into that, this is how they look:

  
  ![image](https://user-images.githubusercontent.com/20567503/209168109-e76d1fca-c346-4461-be17-487304c9fc0d.png)